builder/manager.go

```diff
@@ -320,7 +320,7 @@ func (m *Manager) doLock(path, opType string) error {
 // SigIntCleanup will take care of cleaning up the build process.
 func (m *Manager) SigIntCleanup() {
 	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, os.Interrupt)
+	signal.Notify(ch, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
`os.Interrupt` and `syscall.SIGINT` are the same signal. `os.Interrupt` in fact is defined as `const Interrupt Signal = syscall.SIGINT`.

Reply:
Thanks! :)